src/youdo/state.clj

```diff
@@ -1,10 +1,12 @@
-(ns youdo.state)
+(ns youdo.state
+  (:require [youdo.db :as db]))
 
 (defn list-tasks
   "Given a file path, returns list of tasks."
-  [path])
+  [path]
+  (db/read! path))
```

Comment:
the `!` at the end means the given function affects the outside world. Technically speaking, `read` just reads the data so it doesn't change anything. I think it should be called `read` instead.

Reply:
Unfortunately,  ```read``` is already a clojure function. I will use ```read-content``` instead.